src/compiler/diagnosticMessages.json

```diff
@@ -3085,6 +3086,15 @@
         "category": "Error",
         "code": 18001
     },
+    "No input files were found in config file: '{0}'. 'files' list is empty.": {
+        "category": "Error",
+        "code": 18002
+    },
+    "No input files were found in config file: '{0}'. Here are the contents of include: {1} and exclude: {2} lists.": {

```

Comment:
`No inputs were found in config file '{0}'. Specified 'include' paths were '{1}' and 'exclude' paths were '{2}'.`